### DISCUSSION TITLE
Allow any volume-name in additional volumes for borg-backup

### DISCUSSION THREAD

**Discussion by loeffelpan** [nextcloud/all-in-one#2203] [Ideas] [answered]
I dig into you code and tried to find out why you trim unwanted chars from additional backup string.
https://github.com/nextcloud/all-in-one/blob/0b15352132ac95ffa63d87ff9c27b2654ff4ffec/php/src/Data/ConfigurationManager.php#L667
I have a volume with a "." in its name and would like to add that to the addional volumes for borg backup.
Can we maybe remove that trim? I didn't get why you only allow that regex match.

**Comment by szaimen** [comment, marked as answer]
Hi, dots will be allowed with https://github.com/nextcloud/all-in-one/pull/2207. The regex check is just an additional safeguard so that things dont bug out later on.

**Reply by loeffelpan** [reply]
Thanks. Don't forget to change allowed characters in the description.
https://github.com/nextcloud/all-in-one/blob/fc56c262c14573fb6d2696b6c47c7143620318e4/php/templates/containers.twig#L444

**Reply by szaimen** [reply]
indeed, thanks

**Reply by loeffelpan** [reply]
Any plans to release this (as beta)?

**Reply by szaimen** [reply]
Yes, as soon as I get to it

**Reply by szaimen** [reply]
This is now released with v4.8.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by loeffelpan** [reply]
Working, thanks.
